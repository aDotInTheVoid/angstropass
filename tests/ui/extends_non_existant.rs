#[nanopass::langs]
mod langs {
    mod base {}

    #[extends(base)]
    mod good {}

    #[extends(nonexistant)]
    mod bad {}
}

fn main() {}
