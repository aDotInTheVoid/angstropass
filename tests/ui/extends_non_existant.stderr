error: unknown language: nonexistant
 --> tests/ui/extends_non_existant.rs:8:15
  |
8 |     #[extends(nonexistant)]
  |               ^^^^^^^^^^^
